dev/stage-binaries.sh

```diff
@@ -20,12 +20,12 @@
 
 SCALA_VERSION=2.12
 FLINK_VERSIONS=1.13,1.14,1.15
-SPARK_VERSIONS=2.4,3.0,3.1,3.2
+SPARK_VERSIONS=2.4,3.0,3.1,3.3
```

Comment:
Should add 3.3 and not replace 3.2

Reply:
same as https://github.com/apache/iceberg/pull/5094#discussion_r904705128